lib/cdo/infra_test_topic.rb

```diff
@@ -8,6 +8,16 @@ def self.set_green_commit(commit)
     Slack.update_topic 'infra-test', msg
   end
 
+  # @return [String | nil] Returns the commit specified as :greenbeer: in the Slack#infra-test
+  #   topic (if one exists) or nil.
+  def self.green_commit
+    current_topic = Slack.get_topic('infra-test')
+    unless current_topic =~ /:greenbeer:/
```

Comment:
Is there a style preference between this and `return nil unless current_topic =~ /:greenbeer:/`?